### PR TITLE
Running certbot + nginx

### PR DESCRIPTION
**Pull request by gotibhai**
Fingers crossed.

### CODE REVIEW

File: dc-dev.yml [hunk 1]
```diff
@@ -17,17 +17,16 @@ services:
   webapp_client:
     build: webapp/reactclient
     ports:
-      - "80:80"
+      - "3000:3000"
```
Comment (gotibhai) [inline dc-dev.yml:20]: Changed this file to be the dev file.

File: server/S&P500-Info.csv [hunk 2]
```diff
@@ -227,266 +227,266 @@
 225,HAS,Hasbro Inc.,reports,Consumer Discretionary,Leisure Products,"Pawtucket, Rhode Island",1984-09-30,46080,
 226,HCA,HCA Healthcare,reports,Health Care,Health Care Facilities,"Nashville, Tennessee",2015-01-27,860730,
 227,PEAK,Healthpeak Properties,reports,Real Estate,Health Care REITs,"Long Beach, California",2008-03-31,765880,
-228,HP,Helmerich & Payne,reports,Energy,Oil & Gas Drilling,"Tulsa, Oklahoma[5]",2010-03-01,46765,
```
Comment (gotibhai) [inline server/S&P500-Info.csv:230]: Not sure why these keep changing, Can we add a task to stop this?

File: webapp/nodeServer/expressServer.js [hunk 3]
```diff
@@ -26,13 +26,6 @@ app.use(express.urlencoded());
 
 app.use(index);
 
-//Setup Sockets
-// const credentials = {
-//   key: fs.readFileSync('./certs/privkey.pem'),
-//   cert: fs.readFileSync('./certs/cert.pem'),
-//   cacert: fs.readFileSync('./certs/chain.pem')
```
Comment (gotibhai) [inline webapp/nodeServer/expressServer.js:33]: No longer need this.

File: webapp/nodeServer/routes/index.js [hunk 4]
```diff
@@ -4,11 +4,11 @@ const redis = require("redis");
 //Redis Client
 const client = redis.createClient(6379, host='redis_db');
 
-router.get("/", (req, res) => {
+router.get("/api", (req, res) => {
```
Comment (gotibhai) [inline webapp/nodeServer/routes/index.js:7]: Changing routes now as we added a path for this in the nginx conf.

File: webapp/reactclient/package.json [hunk 5]
```diff
@@ -13,7 +13,7 @@
     "victory": "^34.3.5"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "set HTTPS=true&&react-scripts start",
```
Comment (gotibhai) [inline webapp/reactclient/package.json:16]: This didn't really help, I'll remove it next PR.

File: webapp/reactclient/package.json [hunk 6]
```diff
@@ -33,6 +33,6 @@
       "last 1 safari version"
     ]
   },
-  "proxy": "http://127.0.0.1:5000/",
+  "proxy": "http://localhost:5000/",
```
Comment (gotibhai) [inline webapp/reactclient/package.json:36]: Need this for local only!

File: webapp/reactclient/src/global.js [hunk 7]
```diff
@@ -2,16 +2,14 @@ import socketIOClient from "socket.io-client";
...
 const DEV_HOST = "http://localhost"
-const PROD_HOST = "35.236.122.33"
-if (process.env.REACT_APP_ENV && process.env.REACT_APP_ENV === "PROD") {
-	global.HOST = `${PROD_HOST}:${PORT}`
-} else {
-	// if env is not set; we'll default to PROD
+//const PROD_HOST = "https://35.236.122.33"
+if (process.env.REACT_APP_ENV === "DEV") {
 	global.HOST = `${DEV_HOST}:${PORT}`
+	global.socket = socketIOClient(global.HOST, { transports: ['websocket'] });
```
Comment (gotibhai) [inline webapp/reactclient/src/global.js:8]: Local needs a host, But on prod it just calls on the domain `fofx.xyz` which then after routing from nginx comes to node.

File: server/S&P500-Info.csv (hunk 2 above)
Comment (abdulmi) [reply server/S&P500-Info.csv:230]: Its coz s&p list keeps changing

File: webapp/nodeServer/routes/index.js (hunk 4 above)
Comment (abdulmi) [reply webapp/nodeServer/routes/index.js:7]: Not following, why do we need this path?

**Pull request merged**